common/decklist.cpp

```diff
@@ -555,21 +613,37 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         rx.setPattern("\\|.*$");
         line.remove(rx);
 
+        // If the user inputs "Quicksilver Elemental" then it will cut it off
```

Comment:
The actual fix is in this file. Cleanup was done on all files touched to meet new coding standards.